tests/baselines/reference/lastPropertyInLiteralWins.errors.txt

```diff
@@ -15,16 +14,13 @@ tests/cases/compiler/lastPropertyInLiteralWins.ts(14,5): error TS1117: An object
     }
     test({ // Should error, as last one wins, and is wrong type
         thunk: (str: string) => {},
-        ~~~~~

```

Comment:
This error is gone thanks to the `seen`-related logic that has been added to `elaborateElementwise`. I think it makes sense only to report the error on the last property when dealing with the duplicates as it's the only one that "decides" about the actual final type of this property - if that type is incorrect then the error should be reported there.

Previous duplicates can also have an incorrect type but I think it's a fair tradeoff. Note that previously the error was reported for the "final" type as well, so we could see confusing things like this:
```ts
interface Thing {
    thunk: (str: string) => void;
}
function test(thing: Thing) {
    thing.thunk("str");
}
test({
    thunk: (str: string) => {}, // Type '(num: number) => void' is not assignable to type '(str: string) => void'.
    thunk: (num: number) => {} // Type '(num: number) => void' is not assignable to type '(str: string) => void'.
});
```

Reply:
Since we have both `An object literal cannot have multiple properties with the same name.` and `'a' is specified more than once, so this usage will be overwritten.` errors, I think it usually doesn't matter, one way or another, most of the time. The only case where multiple "effective" instances of a property can coexist is with a spread that includes `undefined` in its' type, so something like
```ts
interface A {
    a?: string
}

declare var a: A;

const b: {a: boolean} = {a: 12, ...a};
```
in which case _both_ instances need to be synchronized to the correct type, not _just_ the last one. So, in deferrence to the only case where there aren't other errors that make having the extra error matter less, I think displaying the (potential) error on all members with the name is relevant.

I want the above example to have a squiggle on both the `a` prop _and_ the `a` spread, since both contribute to the final prop type and both help make it fail, not one or the other. If you just remove that new "seen" logic, you should get to that, and I don't think the "extra" errors in other cases matter much, since other errors also pop up.